### PR TITLE
Add new plugin: Locations and Areas

### PR DESCRIPTION
**Pull request by 100wunder**
I added my WordPress Plugin "Locations and Areas" to reach a bigger audience. It extends basic map functionality by adding Areas as additional map navigation. It is also possible to allow website visitors adding their own locations directly from the frontend. The WordPress community needs more Leaflet.js!

### CODE REVIEW

**Review by jonkoops — APPROVED** [review]
Looks good but there are some conflicts that will need to be resolved.

**Comment by 100wunder** [comment]
@jonkoops I fixed the merge conflict. Thanks for your support!

**Review by Falke-Design — APPROVED** [review]

File: docs/plugins.md [hunk 1]
```diff
@@ -4993,13 +4993,26 @@ The following plugins integrate Leaflet into third party services or websites.
...
   </tr>
   <tr>
-	<td>
-		<a href="https://wordpress.org/plugins/open-user-map/">Open User Map – Users can add locations from the frontend</a>
-	</td><td>
-		WordPress plugin to let your visitors add locations directly from the frontend - without registration. They drop a marker on the map and provide some location details. After submit the location proposal will be “pending” and wait for your review approval to get published.
-	  </td><td>
-		<a href="https://www.open-user-map.com/">100plugins</a>
-	</td>
+    <td>
```
Comment (Falke-Design) [inline docs/plugins.md:4996]: Please replace all blanks with tabs
```suggestion
	<td>
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]

File: docs/plugins.md [hunk 2]
```diff
@@ -4992,13 +4992,26 @@ The following plugins integrate Leaflet into third party services or websites.
 		<a href="https://wordpress.org/plugins/wp-trip-summary/">Alexandru Boia</a>
 	</td>
   </tr>
+  <tr>
```
Comment (johnd0e) [inline docs/plugins.md:4995]: Tabs

File: docs/plugins.md [hunk 3]
```diff
@@ -4992,13 +4992,26 @@ The following plugins integrate Leaflet into third party services or websites.
...
 	<td>
 		<a href="https://wordpress.org/plugins/open-user-map/">Open User Map – Users can add locations from the frontend</a>
-	</td><td>
+	</td>
+	<td>
 		WordPress plugin to let your visitors add locations directly from the frontend - without registration. They drop a marker on the map and provide some location details. After submit the location proposal will be “pending” and wait for your review approval to get published.
-	  </td><td>
-		<a href="https://www.open-user-map.com/">100plugins</a>
+	</td>
+    <td>
```
Comment (johnd0e) [inline docs/plugins.md:5013]: Tabs

**Review by johnd0e — CHANGES_REQUESTED** [review]

File: docs/plugins.md [hunk 4]
```diff
@@ -4941,79 +4941,76 @@ The following plugins integrate Leaflet into third party services or websites.
...
+		</td><td>
+			<a href="https://wordpress.org/plugins/wp-trip-summary/">Alexandru Boia</a>
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<a href="https://wordpress.org/plugins/locations-and-areas/">Locations and Areas – Leaflet Map with Region Tabs</a>
+		</td><td>
+			WordPress plugin to showcase widely distributed locations on a single map with additional navigation tabs for regions. The map is based on Leaflet JS and offers you several free map styles. Gutenberg Block included.
+		</td><td>
```
Comment (Falke-Design) [inline docs/plugins.md:5001]: Please move `<td>` to a new line

File: docs/plugins.md [hunk 5]
```diff
@@ -4941,79 +4941,76 @@ The following plugins integrate Leaflet into third party services or websites.
...
+			A WordPress trip summary plugin to help travel bloggers manage and display structured information about their train rides and biking or hiking trips.
+		</td><td>
+			<a href="https://wordpress.org/plugins/wp-trip-summary/">Alexandru Boia</a>
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<a href="https://wordpress.org/plugins/locations-and-areas/">Locations and Areas – Leaflet Map with Region Tabs</a>
+		</td><td>
```
Comment (Falke-Design) [inline docs/plugins.md:4999]: Please move `<td>` to a new line

File: docs/plugins.md [hunk 6]
```diff
@@ -4941,79 +4941,76 @@ The following plugins integrate Leaflet into third party services or websites.
...
+	<tr>
+		<td>
+			<a href="https://github.com/mwasil/Leaflet.Facebook/">Leaflet.Facebook</a>
+		</td><td>
+			Simple plugin for adding Facebook like button as a control.
+	  	</td><td>
+			<a href="https://marcinwasilewski.eu/">Marcin Wasilewski</a>
+		</td>
+  	</tr>
+  	<tr>
```
Comment (Falke-Design) [inline docs/plugins.md:4987]: Use tabs

File: docs/plugins.md [hunk 7]
```diff
@@ -4941,79 +4941,76 @@ The following plugins integrate Leaflet into third party services or websites.
...
+	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/mwasil/Leaflet.Facebook/">Leaflet.Facebook</a>
+		</td><td>
+			Simple plugin for adding Facebook like button as a control.
+	  	</td><td>
+			<a href="https://marcinwasilewski.eu/">Marcin Wasilewski</a>
+		</td>
+  	</tr>
```
Comment (Falke-Design) [inline docs/plugins.md:4986]: Use tabs

File: docs/plugins.md [hunk 8]
```diff
@@ -4941,79 +4941,76 @@ The following plugins integrate Leaflet into third party services or websites.
...
-	<td>
-		<a href="https://wordpress.org/plugins/locations-and-areas/">Locations and Areas – Leaflet Map with Region Tabs</a>
-	</td>
-	<td>
-		WordPress plugin to showcase widely distributed locations on a single map with additional navigation tabs for regions. The map is based on Leaflet JS and offers you several free map styles. Gutenberg Block included.
-	</td>
-	<td>
-		<a href="https://www.locations-and-areas.com/?ref=leafletjs.com">100plugins</a>
-	</td>
+    	</td>
```
Comment (Falke-Design) [inline docs/plugins.md:4976]: Use tabs

File: docs/plugins.md [hunk 9]
```diff
@@ -4941,79 +4941,76 @@ The following plugins integrate Leaflet into third party services or websites.
...
+					<a href="https://github.com/astridx/pkg_aggpxtrack">Gibhub</a><br />
+					<a href="https://extensions.joomla.org/extensions/extension/maps-a-weather/maps-a-locations/aggpxtrack/">Joomla Extension Directory</a><br />
+				</li>
+				<li>
+					<strong>Agosmmapwithmarker: </strong><br />
+					Custom field for show a map with a marker in frond end - always the right card for the content. You can enter the address in backend.<br />
+					<a href="https://github.com/astridx/plg_fields_agosmmapwithmarker">Gibhub</a><br />
+					<a href="https://extensions.joomla.org/extensions/extension/maps-a-weather/maps-a-locations/agosmmapwithmarker/">Joomla Extension Directory</a><br />
+				</li>
+    		</ul>
```
Comment (Falke-Design) [inline docs/plugins.md:4973]: Use tabs

**Review by Falke-Design — CHANGES_REQUESTED** [review]

**Review by 100wunder — COMMENTED** [review]
@Falke-Design Thanks for your suggestions! I added consistency with tabs and table structure before already and currently wait for @johnd0e approval. It's always possible to optimize further but IMO this should be topic of a future PR.

**Comment by Falke-Design** [comment]
@100wunder you need to change your PR. You can replace the blanks with tabs or you remove the changes that are not necessary for you.

**Comment by 100wunder** [comment]
@Falke-Design okay I did

**Comment by 100wunder** [comment]
@johnd0e If you find some time would you please review? I had send you a review request.

**Comment by Falke-Design** [comment]
We added the plugin in the refactoring where we moved the plugins into seperate files #7805

**Comment by 100wunder** [comment]
Thanks @Falke-Design, @jonkoops and @johnd0e for your contributions and support.

**Pull request closed without merging**